settings.gradle

```diff
@@ -61,6 +61,9 @@ include ':agent:agent-tooling'
 include ':agent:agent-for-testing'
 include ':agent:instrumentation:applicationinsights-web-2.3'
 include ':agent:instrumentation:azure-functions'
+include ':agent:instrumentation:netty-4.0'
```

Comment:
should we drop netty-4.0?  data is from 2021-10 to 2022-4.
I find little usage of it via statsbeat. most users are on netty-4.1.

![image](https://user-images.githubusercontent.com/56097766/168391554-e6adf325-5d5b-43a8-a800-2fd787da17f7.png)

![image](https://user-images.githubusercontent.com/56097766/168391620-05d554f1-e479-4625-9203-e03a1846f54b.png)